src/gen/lib/AST_grammar_conv.ml

```diff
@@ -0,0 +1,94 @@
+(*
+   Conversion and simplification from type specified in Tree_sitter.atd.
+*)
+
+open AST_grammar
+
+(*
+   Simple translation without normalization. Get rid of PREC_*
+*)
+let rec translate (x : Tree_sitter_t.rule_body) =
+  match x with
+  | SYMBOL ident -> Symbol ident
+  | STRING s -> String s
+  | PATTERN s -> Pattern s
+  | BLANK -> Blank
+  | REPEAT x -> Repeat (translate x)
+  | REPEAT1 x -> Repeat1 (translate x)
+  | CHOICE [x; BLANK] -> Optional (translate x)
```

Comment:
I think you can have multiple elements here, not just one 'x'.
See what I wrote there: https://github.com/returntocorp/ocaml-tree-sitter/blob/to_menhir/lib/normalize_grammar2.ml 
and especially this function: https://github.com/returntocorp/ocaml-tree-sitter/blob/c71acb393973081f4f512e2e0a0332a427095e5a/lib/normalize_grammar2.ml#L55

Reply:
In my current implementation, `Blank` can remain, and it maps to `unit`. In the context of a choice, it becomes something like `` `Case5 ()` (although just `` `Case5`` would be nicer).